browser/static/js/main_track.js

```diff
@@ -838,7 +840,7 @@ function prepare_canvas() {
 function action(action, info, frame = current_frame) {
   $.ajax({
     type:'POST',
-    url:`${document.location.origin}/action/${project_id}/${action}/${frame}`,
+    url:`${document.location.origin}/edit/${project_id}/${action}/${frame}`,
```

Comment:
Was this a missed bugfix?

Reply:
Yes, this was missed from the undo PR. The action route was renamed to edit